app/src/main/java/com/infomaniak/drive/ui/LaunchActivity.kt

```diff
@@ -40,29 +42,28 @@ class LaunchActivity : AppCompatActivity() {
 
             logoutCurrentUserIfNeeded() // Rights v2 migration temporary fix
 
-            when {
+            val clazz = when {
```

Comment:
Rename to class

Reply:
I can't do it as class is a reserved keyword